app/views/ExposureHistory/ExposureDatumDetail.tsx

```diff
@@ -126,6 +123,27 @@ const NoDataExposureDetail = ({
   );
 };
 
+interface NextStepsButtonProps {
```

Comment:
Should we move this component to a new file?

Reply:
It's a pretty small change and since there are two more components in this file I figured it is ok to leave it here for now.